### DISCUSSION TITLE
Sailing mode show up, but battery stays charged to the limit.

### DISCUSSION THREAD

**Discussion by JJM-TFG** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1029] [Q&A] [answered]
As the title says, after sailing mode kicks in, the battery just stays put on the stablished limit.

I have it set to 70%, with 10% sailing, and it stays fixed at 70%.
Am I doing anything wrong?
I have tried letting it charge above limit and letting it discharge to see if it kicks in, and the opposite, letting the battery drain below limit and then plugging the macbook in.
I also checked and I am on the latest version, 1.22.2.

Picture for reference:
![image](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/32578793/db4b7799-6032-490b-8e72-aff7f9ba36a4)

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @HykeRG ,

This is actually the intended behavior of the Sailing Mode and looking at your screenshots, Sailing Mode is working as intended.
It is a common misconception that the Sailing Mode does actively charge and discharge your MacBook between the sailing range. This would actually be much worse for the battery.
You can read a detailed description of what the Sailing Mode actually does on our blog "[Feature Explanation: Sailing Mode](https://apphousekitchen.com/feature-explanation-sailing-mode/)"

Best,
Matthias

**Reply by JJM-TFG** [reply]
OHH, I understood it would sail between the Limit and the sailing threshold, not that it bypassed the battery, only kicking in when only the adapter was not enough. Thanks for clarifying that.
